tests/cpp/relay_transform_sequential.cc

```diff
@@ -70,7 +70,7 @@ TEST(Relay, Sequential) {
   auto mod = IRModule::FromExpr(func);
   auto pass_ctx = relay::transform::PassContext::Create();
   pass_ctx->opt_level = 3;
-  pass_ctx->fallback_device = 1;
+  pass_ctx->config.Set("relay.fallback_device_type", IntImm(DataType::Int(32), 1));
```

Comment:
After merging the code, I find that we can further simplify it to pass_ctx->config.Set("relay.fallback_device_type", Integer(1));

Reply:
Yeah, I tried this but the compilation failed. I can take a deeper look though.